.travis.yml

```diff
@@ -6,14 +6,23 @@ script: bundle exec rake
 rvm:
   - 1.9.3
   - 2.1.6
+  - 2.2.6
+  - 2.3.3
+  - 2.4.0
 env:
   - PUPPET_VERSION="~> 3.0.0"
   - PUPPET_VERSION="~> 3.1.0"
   - PUPPET_VERSION="~> 3.2.0"
   - PUPPET_VERSION="~> 3.4.0"
   - PUPPET_VERSION="~> 3.7.3"
+  - PUPPET_VERSION="~> 3.8.7"
```

Comment:
I think we should only test against the newest in each major tree. So in this case ~ `3.8.7` and ~ `4.8.0`